programs/kbuild/parse

```diff
@@ -1,19 +1,45 @@
 #!/usr/bin/env ruby
 
```

Comment:
there's no change of programs/kbuild/run, how can it generate these output?

Also the check fails, you need resolve it as well.

Reply:
ok，since I have run an empty run_kbuild function locally, I've only uploaded the effect of adding time to the kbuild function, which is not the version I'm running locally.

Reply:
this is ok, i see the change of run is commited by yujie.